src/components/Profile.js

```diff
@@ -1,33 +1,46 @@
 import { Avatar, Box, Typography } from '@mui/material';
 import { useAuthContext } from '~/contexts/hooks';
 import useImage from '~/hooks/useImage';
+import theme from '~/theme';
 
 const DEFAULT_IMAGE_URL = '/images/logo.png';
-export default function Profile(props) {
+
+export default function Profile({ ...props }) {
   const { session } = useAuthContext();
   const imageUrl = useImage(session?.imgId);
+
   return (
     <Box sx={{ display: 'flex', gap: 1 }}>
       {/* Full Name */}
-      <Typography sx={{ justifyContent: 'center', alignContent: 'center' }}>
+      <Typography
+        sx={{ justifyContent: 'center', alignContent: 'center', fontSize: 12 }}
+      >
         {session
-          ? `${session.firstName} ${session.middleName} ${session.lastName}`
+          ? `${session.lastName} ${session.middleName} ${session.firstName}`
```

Comment:
null check session fields, session might not be null, but its properties could be